incubator/kafka/templates/statefulset.yaml

```diff
@@ -51,7 +51,7 @@ spec:
         command:
         - sh
         - -c
-        - "./bin/kafka-server-start.sh config/server.properties --override zookeeper.connect={{ .Release.Name }}-zookeeper:2181/ --override broker.id=${HOSTNAME##*-}"
+        - "./bin/kafka-server-start.sh config/server.properties --override zookeeper.connect={{ .Release.Name }}-zookeeper:2181/ --override log.dirs={{ printf "%s/logs" .Values.DataDirectory | trunc 63 }} --override broker.id=${HOSTNAME##*-}"
```

Comment:
should we not now move this to a config-map or something?

Reply:
`--override log.dirs={{ printf "%s/logs" .Values.DataDirectory | trunc 63 }}`
I'm just wondering, is there any reason to truncate the path? As I understand, we might run into hard-to-debug problems when `DataDirectory` would be long.

If there's no particular reason, let me know, I will submit a PR.

Reply:
@KarolisL , this change was done in #1447 and got lost when #1321 was merged. I've just restored it. Maybe original author @prune998 can tell.

Reply:
no real reason to truncate the path... truncating or path too long will both end with an error :)